src/tests/stats/test_mean_f03.f90

```diff
@@ -5,34 +5,51 @@ program test_mean
 use stdlib_experimental_stats, only: mean
 implicit none
 
+real(dp), parameter :: dptol = 2.2e-15_dp
```

Comment:
This number seems to be `10 * epsilon(1._dp)`. Maybe you can use that instead.

Reply:
(Or you can keep it and we'll fix it in another PR.)

Reply:
I just sent a PR for this: #142.

Reply:
I did not see your message. I corrected it both in test_mean_f03.f90 and test_mean.f90

Reply:
No problem. Let's first get an agreement on this in #142, to ensure nobody is against it.